design/FY2017/NFP-SingleDuctAirTerminalConstantVolumeNoReheat.md

```diff
@@ -0,0 +1,212 @@
+
+![](ATSDCVNoReheat.png)
+
+                         Figure 1 Air terminal single duct constant volume without reheat
+
+
+
+## IDD - Input Data Dictionary ##
+This new IDD object has all input fields of the *AirTerminal:SingleDuct:Uncontrolled* IDD object except that the *Design Specification Air Terminal Sizing Object Name* input field. The value for the later input field can be obtained from the parent object, ZoneHVAC:AirDistributionUnit. Also unlike the *Uncontrolled* object the new object has an inlet and an outlet air nodes input fields. Proposed idd object is shown below.
+
+```
+
+AirTerminal:SingleDuct:ConstantVolume:NoReheat,
+
+       \memo Central air system terminal unit, single duct, constant volume, without reheat coil
+
+  A1 , \field Name
+
+       \required-field
+       \reference AirTerminalUnitNames
+
+  A2 , \field Availability Schedule Name
+
+       \note Availability schedule name for this system. Schedule value > 0 means the system is available.
+       \note If this field is blank, the system is always available.
+       \type object-list
+       \object-list ScheduleNames
+
+  A3,  \field Air Outlet Node Name
+
+       \required-field
+       \type node
+       \note This is an air outlet node from the air distribution unit. This node name should be one of the 
+       \note supply air inlet node names of a zone served by this component.
+
+  A4,  \field Air Inlet Node Name
+
+       \required-field
+       \type node
+       \note The air-inlet node name that connects the air splitter to the individual zone air distribution 
+       \note unit. This node should also be one of the outlet air node of an AirLoopHVAC:ZoneSplitter component.
+
+  N1,  \field Maximum Air Flow Rate
+
+       \required-field
+       \units m3/s
+       \minimum 0.0
+       \autosizable
+
+   A5, \field Design Specification Outdoor Air Object Name
+
+       \type object-list
+       \object-list DesignSpecificationOutdoorAirNames
+       \note When the name of a DesignSpecification:OutdoorAir object is entered, the terminal
+       \note unit will adjust flow to meet this outdoor air requirement and no more. Load is still "Uncontrolled."
+       \note If Outdoor Air Flow per Person is non-zero, then the outdoor air requirement will
+       \note be computed based on the current number of occupants in the zone, as for demand controlled ventilation.
```

Comment:
"be computed based on the on the current number of occupants in the zone, as for demand controlled ventilation." --> "be computed based on either the current or design occupancy as specified in the Per Person Ventilation Rate Mode field."

Reply:
Ok. will correct this statement.